src/NuGetGallery/Scripts/gallery/page-edit-readme.js

```diff
@@ -336,6 +336,10 @@ var BindReadMeDataManager = (function () {
             if (response.ImagesRewritten) {
                 displayReadMeWarning("Some images were automatically rewritten to use secure links and might be broken.");
             }
+
+            if (response.ImageSourceDisallowed) {
+                displayReadMeWarning("Some images were broken might due to images source are not from trusted services.");
```

Comment:
@chgill-MSFT Could you help with the wording here, aslo I think it will be great if we have a page that customer could refer to about details info for allowlist

Reply:
What do you think of this?

```suggestion
                displayReadMeWarning("Some images are not displayed as they are not from trusted domains.");
```

Reply:
Will we have a docs page that explains this in more detail? Could we link to that?

Reply:
Yes, that's a plan to have a link with more details like what we did for license deprecation details url.

Reply:
I think Loic's suggested wording and the idea to link to a doc with the trusted domains list is a great idea! I'll start drafting some documentation so we can include links in the warnings. I have an issue tracking the docs here: https://github.com/NuGet/Product/issues/43

Reply:
@chgill-MSFT , the issue you linked it returns 404 for me. it because of permission?